src/main/resources/application-oauth.properties

```diff
@@ -0,0 +1,28 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Huawei Inc.
+#
+#
+# spring security oauth2 config
+logging.level.org.springframework.web=info
+logging.level.org.springframework.security=debug
+# set authorization-token-type: JWT or OpaqueToken
+authorization-token-type=OpaqueToken
+# set authorization server endpoint and client configs
+authorization-server-endpoint=https://iam.xpanse.site
```

Comment:
to keep terraform-boot independent, we must remove these config values.

Reply:
removed